### PR TITLE
zotero: add page

### LINKED ISSUE #10189: page request: zotero
### Command description

[Zotero](https://www.zotero.org/) is a bibliography manager. Even though it is a GUI application, it has several command-line options (eg for starting it in headless mode, for changing its behaviour etc.)

As I'm no expert on these options, I'd welcome if a knowledgable person could add a tldr page for Zotero.

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [X] macOS
- [ ] sunOS
- [X] Windows

### PR DESCRIPTION
**Pull request by dch82**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 6.0.26

---

Closes #10189.

### CODE REVIEW

File: pages/common/zotero.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Zotero
```
Comment (mfrw) [inline pages/common/zotero.md:1]:
```suggestion
# zotero
```

File: pages/common/zotero.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+> Manage your bibliographies.
+> More information: <https://www.zotero.org/support>.
+
+- Run Zotero with the GUI:
+
+`zotero`
+
+- Run Zotero headlessly:
```
Comment (mfrw) [inline pages/common/zotero.md:10]: Taking inspiration from `firefox.md`
```suggestion
- Run Zotero in headless mode:
```

File: pages/common/zotero.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`zotero --headless`
+
+- Run Zotero with a particular profile:
```
Comment (mfrw) [inline pages/common/zotero.md:14]:
```suggestion
- Run Zotero with a specific profile:
```

**Review by mfrw — COMMENTED** [review]
Thank you for your contribution @dch82.
I have a few cosmetic nit-picks :)

File: pages/common/zotero.md (hunk 1 above)
Comment (dch82) [reply pages/common/zotero.md:1]: I actually think it should remain capitalised as it is not just a command but a platform by itself

File: pages/common/zotero.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# zotero
+
+> Manage your bibliographies.
+> More information: <https://www.zotero.org/support>.
+
+- Run Zotero with the GUI:
```
Comment (kbdharun) [inline pages/common/zotero.md:6]:
```suggestion
- Run with the GUI:
```

File: pages/common/zotero.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`zotero`
+
+- Run Zotero in headless mode:
```
Comment (kbdharun) [inline pages/common/zotero.md:10]:
```suggestion
- Run in headless mode:
```

File: pages/common/zotero.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`zotero --headless`
+
+- Run Zotero with a specific profile:
```
Comment (kbdharun) [inline pages/common/zotero.md:14]:
```suggestion
- Run with a specific profile:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Since Zotero is already present in the title, I don't think we need to repeat it in every command description.

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**